### PR TITLE
Adds Slack notifications about releases, upgrades to nextmv-py v0.11.1

### PR DESCRIPTION
**Pull request by merschformann**
# Description

Added Slack notifications for improving internal transparency about ongoing releases.

## Changes

- Added _optional_ `--slack-url` parameter to release workflow command
- Added `notify_slack` function to send release notifications to Slack
- Included `requests>=2.26.0` in `requirements.txt` for HTTP requests to Slack Webhooks
- Upgrades to `nextmv-py==v0.11.1`

### CODE REVIEW

File: .nextmv/release/main.py [hunk 1]
```diff
@@ -241,5 +251,21 @@ def bump_marketplace_version(
     return f"v{major}.{minor}.{patch}"
 
 
+def notify_slack(url: str, app: str, version: str, marketplace_version: str):
```
Comment (sebastian-quintero) [inline .nextmv/release/main.py]: The return type hint should be None

File: .nextmv/release/main.py [hunk 2]
```diff
@@ -241,5 +251,21 @@ def bump_marketplace_version(
...
+    """
+    Notify Slack channel about release.
+    """
+
+    response = requests.post(
+        url,
+        json={
+            "text": f"Release notification - community-apps/{app} {version} / marketplace {marketplace_version}",
```
Comment (sebastian-quintero) [inline .nextmv/release/main.py:262]: I think we should either only notify for prod or log the environment that is being released in here.

**Review by sebastian-quintero — APPROVED** [review]
Non-blocking comments

File: .nextmv/release/main.py (hunk 2 above)
Comment (merschformann) [reply .nextmv/release/main.py:262]: Yeah, totally. I somehow missed the distinction here. The parameter is optional, so, I will simply drop it in the workflow if not targeting prod. :relaxed:

Comment (merschformann) [reply .nextmv/release/main.py:262]: Thanks for noticing! :hugs:

**Comment by merschformann** [comment]
This will need a fix for `nextmv-py` for handling dashed parameters before merging. :relaxed:

**Pull request merged**